woocommerce/wc-admin
# Remove the deprecated Card component

Card has been deprecated from `@woocommerce/components` for a long time now, so it's time to remove it altogether. I've also updated examples where it was mentioned.

### Testing instructions

1. Check that building the packages still works: `npm i && npm run clean && npm run build:packages`
2. Smoke test wc-admin still works with the new changes, ie `npm i && npm run clean && npm start` and check for console errors during build and on visiting front-end screens in wc-admin
3. Check that storybook is still ok. `npm run storybook`, smoke test

No changelog required (package specific changelog provided)
